pkg/kv/kvserver/store_raft.go

```diff
@@ -818,12 +818,17 @@ func (s *Store) updateLivenessMap() {
 		// Raft transport, so ConnHealth should usually indicate a real problem if
 		// it gives us an error back. The check can also have false positives if the
 		// node goes down after populating the map, but that matters even less.
-		entry.IsLive = (s.cfg.NodeDialer.ConnHealth(nodeID, rpc.SystemClass) == nil)
+		entry.IsLive = !s.TestingKnobs().DisableLivenessMapConnHealth &&
```

Comment:
The comment above this is mostly stale, thanks to the rpc circuit breakers we will almost always (except when this node first starts up and makes its initial connections) know whether the peer is unreachable or not.

Reply:
Tacked on a commit.